### PR TITLE
Gh-322: Added gremlin gaffer image and compose

### LINKED ISSUE #322: Add docker compose for Gaffer tinkerpop graph
A new docker compose file should be created that starts a [gremlin-server](https://hub.docker.com/r/tinkerpop/gremlin-server) and connects it to a Gaffer graph.

### PR DESCRIPTION
**Pull request by t92549**
# Related Issue

- Resolve #322

### CODE REVIEW

File: docker/gremlin-gaffer/conf/gremlin-server-empty-gaffer.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  gremlin-groovy: {
+    plugins: { org.apache.tinkerpop.gremlin.server.jsr223.GremlinServerGremlinPlugin: {},
+               uk.gov.gchq.gaffer.tinkerpop.gremlinplugin.GafferPopGremlinPlugin: {},
+               org.apache.tinkerpop.gremlin.groovy.jsr223.GroovyCompilerGremlinPlugin: {enableThreadInterrupt: true},
+               org.apache.tinkerpop.gremlin.jsr223.ImportGremlinPlugin: {classImports: [java.lang.Math], methodImports: [java.lang.Math#*]},
+               org.apache.tinkerpop.gremlin.jsr223.ScriptFileGremlinPlugin: {files: [scripts/empty-sample.groovy]}}
+  }
+}
+serializers:
+  - { className: org.apache.tinkerpop.gremlin.driver.ser.GraphBinaryMessageSerializerV1, config: { serializeResultToString: true }}                                                                 # application/vnd.graphbinary-v1.0-stringd
```
Comment (GCHQDeveloper314) [inline docker/gremlin-gaffer/conf/gremlin-server-empty-gaffer.yaml:30]: What's the bit at the end of the line about?

File: docker/gremlin-gaffer/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+Gaffer
```
Comment (GCHQDeveloper314) [inline docker/gremlin-gaffer/README.md:1]: Shouldn't this be Gremlin with Gaffer plugin?

**Review by GCHQDeveloper314 — COMMENTED** [review]
Might be good to mention the notebook in the `README`

File: docker/gremlin-gaffer/conf/gremlin-server-empty-gaffer.yaml (hunk 1 above)
Comment (t92549) [reply docker/gremlin-gaffer/conf/gremlin-server-empty-gaffer.yaml:30]: No idea is part of the default

File: docker/gremlin-gaffer/README.md (hunk 2 above)
Comment (t92549) [reply docker/gremlin-gaffer/README.md:1]: Yes sorry!

**Review by GCHQDeveloper314 — APPROVED** [review]

**Pull request merged**